curriculum/challenges/english/25-front-end-development/workshop-balance-sheet/61fd77f7ad2aeb5ae34d07d6.md

```diff
@@ -107,7 +137,12 @@ assert(document.querySelector('tbody')?.querySelectorAll('tr')?.[1]?.querySelect
             </thead>
             <tbody>
               <tr class="data">
-                <th>Cash <span class="description">This is the cash we currently have on hand.</span></th>
+                <th>
+                  Cash
+                  <span class="description"
+                    >This is the cash we currently have on hand.</span
+                  >
```

Comment:
This looks off, there are multiple. Can you fix this?

Reply:
that's a weirdness of prettier I would guess